Sources/AsyncStream.swift

```diff
@@ -1,7 +1,7 @@
 public protocol AsyncStream {
     var closed: Bool { get }
     func close() -> Bool
-    func receive(length length: Int, result: (Void throws -> Data) -> Void)
-    func send(data data: Data, result: (Void throws -> Void) -> Void)
+    func receive(upTo byteCount: Int, result: (Void throws -> Data) -> Void)
```

Comment:
I'm one such that develops with out an IDE.

Perhaps we could look at using something like maxBytes as a label so that it reads more explicitly.

`receive max bytes in the execution of`